app/persisters/file_set_appending_persister.rb

```diff
@@ -13,17 +13,18 @@ def save(model:)
     files(model).each do |file|
       file_node = create_node(file)
       file_set = create_file_set(file_node)
-      file_identifier = repository.upload(file: file, model: file_node)
-      file_node.file_identifiers = file_node.file_identifiers + [file_identifier.id]
+      file = repository.upload(file: file, model: file_node)
+      file_node.file_identifiers = file_node.file_identifiers + [file.id]
       persister.save(model: file_node)
+      Valkyrie::DerivativeService.for(file_set).create_derivatives
       model.member_ids = model.member_ids + [file_set.id]
     end
     model.try(:sync)
     persister.save(model: model)
   end
 
   def create_node(file)
-    persister.save(model: node_factory.new(label: file.original_filename, original_filename: file.original_filename, mime_type: file.content_type))
+    persister.save(model: node_factory.new(label: file.original_filename, original_filename: file.original_filename, mime_type: file.content_type, use: "original"))
```

Comment:
Is this the same kind of use as pcdm:use? If so, do we want to use the pcdm:use terms/URIs?